lindera/src/tokenizer.rs

```diff
@@ -94,6 +94,14 @@ impl TokenizerBuilder {
         })
     }
 
+     pub fn from_config(config: TokenizerConfig) -> LinderaResult<Self> {
```

Comment:
Can you please format this code?

Reply:
My bad, tried making a commit through the browser for the first time, didn't even contemplate that formatting isn't completed automatically :')